docs/guide/vite.md

```diff
@@ -0,0 +1,383 @@
+
+* Use the built-in alias `Modules/module-name/components/...` for accessing ApostropheCMS module components
+* Project-specific aliases (pointing to your own source code) work as normal using standard Vite alias syntax
+:::
+
+## Development-Specific Features
+
+### Conditional Code Injection
+
+When building UI components, you may want certain elements to appear only during development or when HMR is active. ApostropheCMS's Nunjucks components system makes this easy to manage.
+
+### Component Setup
+
+First, create your component template:
+
+<AposCodeBlock>
+
+```njk
+<div class="dev-tools">
+  <!-- Your development-only UI here -->
+  <div class="dev-tools__status">HMR Active</div>
+  <div class="dev-tools__reload">Force Refresh</div>
+</div>
+```
+  <template v-slot:caption>
+    modules/devTools/views/devTools.html
+  </template>
+
+</AposCodeBlock>
+
+Then, register and configure the component in your module:
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  components(self) {
+    return {
+      devTools(req, data) {
+        return {
+          // Component data here
+          status: 'connected'
+        };
+      }
+    };
+  },
+  init(self) {
+    // Insert the component into the page
+    self.apos.template.prepend({
+      where: 'head',  // or 'body', 'footer', etc.
+      when: 'hmr',    // Show only when HMR is active
+      bundler: 'vite', // Ensure Vite is the active bundler
+      component: 'my-module:devTools'
+    });
+  }
+};
+```
+  <template v-slot:caption>
+    modules/devTools/index.js
+  </template>
+
+</AposCodeBlock>
+
+### Visibility Options
+
+The `when` option controls when your component appears:
+
+```javascript
+when: 'hmr'   // Only visible when HMR is active
+when: 'dev'   // Visible in any development mode
+when: 'prod'  // Only visible in production
+```
+
+You can combine these with `bundler: 'vite'` to ensure your component only appears when using the Vite bundler.
+
+### Common Use Cases
+
+- Development toolbars
+- Debug information
+- Performance monitoring
+- Asset reload buttons
+
+::: warning
+Remember that components marked with `when: 'dev'` or `when: 'hmr'` will never appear in production, regardless of other settings.
+:::
+
+## Migration and Technical Considerations
+
+### Moving from Webpack to Vite
+
+When migrating your ApostropheCMS project to use Vite, you'll need to make a few code adjustments:
+
+1. **Update CSS/Sass Imports**
+   ```scss
+   // Old webpack style
+   @import "~normalize.css";
+   
+   // New Vite style - remove the ~ prefix
+   @import "normalize.css";
+   ```
+
+2. **Update Module Import Paths**
+   ```javascript
+   // Not recommended
+   import Component from 'apostrophe/modules/module-name/ui/apos/components/Component.vue';
+   
+   // Recommended - use the alias
+   import Component from 'Modules/module-name/components/Component.vue';
```

Comment:
One more recommendation I forgot to mention - don't leave out the extension when importing, `.vue`, even `.js` is a good thing most of the time.

Reply:
Added.